WordPress/Classes/ViewRelated/Me/MeViewController.m

```diff
@@ -200,16 +201,16 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             AccountService *accountService = [[AccountService alloc] initWithManagedObjectContext:context];
             WPAccount *defaultAccount = [accountService defaultWordPressComAccount];
 
-            cell.textLabel.textAlignment = NSTextAlignmentCenter;
             cell.accessoryType = UITableViewCellAccessoryNone;
 
             if (defaultAccount) {
-                NSString *signOutString = NSLocalizedString(@"Sign Out", @"Sign out from WordPress.com");
+                NSString *signoutFromAccunt = [NSString stringWithFormat:@"Unlink %@", defaultAccount.username];
+                NSString *signOutString = NSLocalizedString(signoutFromAccunt, @"Sign out from WordPress.com");
```

Comment:
If we change the "linked" verbiage, maybe just say "Sign out from WordPress.com"?  The user's screen name is already visible below the avatar.